lib/PINNBench/Burgers/hpo.py

```diff
@@ -1,54 +1,104 @@
 import numpy as np
+
 from deephyper.problem import HpProblem
 from deephyper.search.hps import CBO
-from .model import get_data, PINN, BurgerSupervisor
+from .model import get_data, PINN, BurgerSupervisor, plotter
+from deephyper.stopper import LCModelStopper
+from deephyper.evaluator import profile, RunningJob
+
+
+@profile
+def run(job: RunningJob) -> dict:
+    # load data
+    nu = 0.01 / np.pi
+    train, val, test = get_data(NT=200, NX=128, X_U=1, X_L=-1, T_max=1, Nu=nu)
+
+    config = job.parameters
+    min_b = 1
+    max_b = 1000
 
-# define the search space
-problem = HpProblem()
-problem.add_hyperparameter((5, 20), "num_layers", default_value=5)
-problem.add_hyperparameter((1e-5, 1e-2), "lr", default_value=0.01)
-problem.add_hyperparameter((5, 50), "hidden_dim", default_value=5)
-problem.add_hyperparameter((200, 300), "epochs", default_value=200)
-problem.add_hyperparameter((0., 5.), "alpha", default_value=.5)
-problem.add_hyperparameter(["relu", "leaky_relu", "tanh", "elu",
-                            "gelu", "sigmoid"],
-                            "activation", default_value='tanh')
-
-
-# define the run function
-def run(config):
-    nu=0.01/np.pi
-    train, val, test = get_data(NT=200, NX=128, X_U=1, X_L=-1, T_max=1,
-                                Nu=nu)
     num_layers = config["num_layers"]
     hidden_dim = config["hidden_dim"]
     output_dim = 1
-    epochs = config["epochs"]
+    epochs = 1  #
```

Comment:
What does this parameter change?

Reply:
With the `LCModelStopper`, the number of epoch for each run is set to 1.

Reply:
But isn't the `LCModelStopper` deciding what number of epochs to run?